mountinfo/mounted_linux.go

```diff
@@ -7,6 +7,50 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// MountedFast is a method of detecting mount points without reading
+// mountinfo from procfs. A mount-point check is guaranteed to be a
+// mount-point or not only when sure is true. When sure is false, the
+// caller needs to check for other methods (eg: parse /proc/mounts)
+// to successfully determine if it is a mount-point.
+func MountedFast(path string) (mounted, sure bool, err error) {
```

Comment:
basically we need three possible state, true, false, and not sure.
does use 1, 0, -1 or for return value a good choice or something like https://stackoverflow.com/questions/314800/best-way-to-define-true-false-unset-state?

Reply:
or if not sure, we can return error indicating cannot figure out whether it is mounted or not. \